src/selections/selection-position.js

```diff
@@ -71,7 +71,15 @@ const imageSelectionSetPosition = function(payload) {
 		: null;
 
 	if (selectionData && selectionData.element) {
-		centerToolbar(this, selectionData.element.getClientRect());
+		var nativeEditor = payload.editor.get('nativeEditor');
+		var uiNode = nativeEditor.config.uiNode;
+
+		var scrollTop = uiNode ? uiNode.scrollTop : 0;
+
+		var rect = selectionData.element.getClientRect();
+		rect.top += scrollTop;
+
+		centerToolbar(this, rect);
```

Comment:
These `var` should be `const` (or `let`, in the case of `let rect`) otherwise we'll be adding new lint errors here.